build/images/Dockerfile.spark-jobs.ubuntu

```diff
@@ -10,10 +10,14 @@ RUN apt-get --allow-releaseinfo-change update && \
     apt-get install -y --no-install-recommends wget ca-certificates && \
     wget https://github.com/ClickHouse/clickhouse-jdbc/releases/download/v0.3.1/clickhouse-jdbc-0.3.1.jar -P /opt/spark/jars/
 
-RUN pip3 install --upgrade pip && \
-    pip3 install pyyaml && \
-    pip3 install kubernetes
-
 COPY plugins/policy-recommendation/policy_recommendation_job.py /opt/spark/work-dir/policy_recommendation_job.py
 COPY plugins/policy-recommendation/policy_recommendation_utils.py /opt/spark/work-dir/policy_recommendation_utils.py
 COPY plugins/policy-recommendation/antrea_crd.py /opt/spark/work-dir/antrea_crd.py
+COPY plugins/anomaly-detection/AnomalyDetection.py /opt/spark/work-dir/AnomalyDetection.py
+COPY plugins/anomaly-detection/requirements.txt /opt/spark/work-dir/anomaly_detector_requirements.txt
```

Comment:
Suggestion: follow our naming convention of python files, `AnomalyDetection.py` -> `anomaly_detection.py`, `anomaly_detector_requirements.txt` -> `anomaly_detection_requirements.txt`.